lighthouse-core/test/lib/network-recorder-test.js

```diff
@@ -249,9 +249,7 @@ describe('network recorder', function() {
       ];
 
       const periods = NetworkRecorder.findNetworkQuietPeriods(records, 0);
-      assert.deepStrictEqual(periods, [
-        {start: 1200, end: Infinity},
-      ]);
+      assert.deepStrictEqual(periods, []);
```

Comment:
what changed with the quiet period?

Reply:
this was the intentional removal of our hack for frame roots, now if an iframe never fires `loadingFinished` it shouldn't be seen as finished

Reply:
ah, got it, I wasn't thinking about the `network-recorder.js` changes